Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/AnalyticsModule.kt

```diff
@@ -5,26 +5,61 @@ import dagger.Provides
 import dagger.multibindings.IntoSet
 import de.rki.coronawarnapp.datadonation.analytics.modules.DonorModule
 import de.rki.coronawarnapp.datadonation.analytics.modules.exposureriskmetadata.ExposureRiskMetadataDonor
-import de.rki.coronawarnapp.datadonation.analytics.modules.exposurewindows.NewExposureWindowsDonor
-import de.rki.coronawarnapp.datadonation.analytics.modules.keysubmission.KeySubmissionStateDonor
-import de.rki.coronawarnapp.datadonation.analytics.modules.registeredtest.RegisteredTestDonor
+import de.rki.coronawarnapp.datadonation.analytics.modules.usermetadata.UserMetadataDonor
+import de.rki.coronawarnapp.datadonation.analytics.server.DataDonationAnalyticsApiV1
+import de.rki.coronawarnapp.datadonation.analytics.storage.DefaultLastAnalyticsSubmissionLogger
+import de.rki.coronawarnapp.datadonation.analytics.storage.LastAnalyticsSubmissionLogger
+import de.rki.coronawarnapp.environment.datadonation.DataDonationCDNHttpClient
+import de.rki.coronawarnapp.environment.datadonation.DataDonationCDNServerUrl
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.converter.protobuf.ProtoConverterFactory
+import javax.inject.Singleton
 
 @Module
 class AnalyticsModule {
 
-    @IntoSet
+    @Singleton
     @Provides
-    fun newExposureWindows(module: NewExposureWindowsDonor): DonorModule = module
+    fun provideAnalyticsSubmissionApi(
```

Comment:
Please align with @chris-cwa regarding the http client for the datadonation server, there are some differences in your PRs.

https://github.com/corona-warn-app/cwa-app-android/pull/2311#discussion_r573655003